project/src/main/world/career-world.gd

```diff
@@ -10,10 +10,32 @@ const X_DIST_BETWEEN_CUSTOMERS := 200
 ## vertical distance separating the customers from the player's path
 const Y_DIST_BETWEEN_CUSTOMERS_AND_PATH := 80
 
+const MOODS_COMMON := [
+	ChatEvent.Mood.SMILE0,
+	ChatEvent.Mood.SMILE1,
+	ChatEvent.Mood.WAVE0,
+	ChatEvent.Mood.WAVE1,
+]
+
+const MOODS_UNCOMMON := [
+	ChatEvent.Mood.LAUGH0,
+	ChatEvent.Mood.LAUGH1,
+	ChatEvent.Mood.LOVE1,
+	ChatEvent.Mood.AWKWARD0,
+]
+
+const MOODS_RARE := [
+	ChatEvent.Mood.AWKWARD1,
+	ChatEvent.Mood.SIGH0,
+	ChatEvent.Mood.SWEAT0,
+	ChatEvent.Mood.THINK0,
+]
+
 export (NodePath) var player_path2d_path: NodePath
 
 ## List of Customer instances for the level's customers. Index 0 holds the leftmost customer.
 var customers := []
+var customer_moods := []
```

Comment:
This should be private and have a comment.

Reply:
Made private and added a comment, `List of moods each customer has when their level is chosen`